src/handlers/guild.ts

```diff
@@ -622,3 +622,100 @@ export function getGuild(guildID: string, counts = true) {
     { with_counts: counts },
   ) as Promise<UpdateGuildPayload>;
 }
+
+/**
+ * Returns the guild template if it exists
+ * @param guildID The ID of the guild
+ * @param code The code of the template to get
+ */
+export function getGuildTemplate(guildID: string, code: string) {
+  const endpoint = `${endpoints.GUILD_TEMPLATES(guildID)}/${code}`;
+  return RequestManager.get(endpoint);
+}
+
+/**
+ * Create a new guild based on a template
+ * NOTE: This endpoint can be used only by bots in less than 10 guilds.
```

Comment:
Can you add a check for this please. If the bot is in 10 guilds throw an error

Reply:
Should it be `cache.guilds.size` to obtain the size of the guilds the bot is in?